py/vttest/mysql_db.py

```diff
@@ -35,3 +35,7 @@ def port(self):
 
   def unix_socket(self):
     raise NotImplementedError('MySqlDB is the base class.')
+
+  def config(self):
+    """Returns the json config to output."""
+    raise NotImplementedError('MySqlDB is the base class.')
```

Comment:
It's not necessary, but it's a good place to document the interface, and if people forget to implement the method, they get a nice message and pointer to the doc too. So I'd rather keep it that way. :)